{
    "dataType": "CVE_RECORD",
    "dataVersion": "5.1",
    "cveMetadata": {
        "cveId": "CVE-2022-43883",
        "assignerOrgId": "9a959283-ebb5-44b6-b705-dcc2bbced522",
        "state": "PUBLISHED",
        "assignerShortName": "ibm",
        "dateReserved": "2022-10-26T15:46:22.831Z",
        "datePublished": "2022-12-19T20:47:46.352Z",
        "dateUpdated": "2024-07-04T11:59:33.794Z"
    },
    "containers": {
        "cna": {
            "affected": [
                {
                    "defaultStatus": "unaffected",
                    "product": "Cognos Analytics",
                    "vendor": "IBM",
                    "versions": [
                        {
                            "status": "affected",
                            "version": "11.1.7, 11.2.0, 11.2.1"
                        }
                    ]
                }
            ],
            "descriptions": [
                {
                    "lang": "en",
                    "supportingMedia": [
                        {
                            "base64": false,
                            "type": "text/html",
                            "value": "\n\n<span style=\"background-color: rgb(204, 217, 226);\">IBM Cognos Analytics 11.1.7, 11.2.0, and 11.2.1 could be vulnerable to a Log Injection attack by constructing URLs from user-controlled data. This could enable attackers to make arbitrary requests to the internal network or to the local file system. IBM X-Force ID: 240266.</span>\n\n"
                        }
                    ],
                    "value": "\nIBM Cognos Analytics 11.1.7, 11.2.0, and 11.2.1 could be vulnerable to a Log Injection attack by constructing URLs from user-controlled data. This could enable attackers to make arbitrary requests to the internal network or to the local file system. IBM X-Force ID: 240266.\n\n"
                }
            ],
            "metrics": [
                {
                    "cvssV3_1": {
                        "attackComplexity": "LOW",
                        "attackVector": "NETWORK",
                        "availabilityImpact": "NONE",
                        "baseScore": 6.5,
                        "baseSeverity": "MEDIUM",
                        "confidentialityImpact": "NONE",
                        "integrityImpact": "HIGH",
                        "privilegesRequired": "LOW",
                        "scope": "UNCHANGED",
                        "userInteraction": "NONE",
                        "vectorString": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:N/I:H/A:N",
                        "version": "3.1"
                    },
                    "format": "CVSS",
                    "scenarios": [
                        {
                            "lang": "en",
                            "value": "GENERAL"
                        }
                    ]
                }
            ],
            "problemTypes": [
                {
                    "descriptions": [
                        {
                            "description": "117 Improper Output Neutralization for Logs",
                            "lang": "en"
                        }
                    ]
                }
            ],
            "providerMetadata": {
                "orgId": "9a959283-ebb5-44b6-b705-dcc2bbced522",
                "shortName": "ibm",
                "dateUpdated": "2022-12-19T20:47:46.352Z"
            },
            "references": [
                {
                    "tags": [
                        "vendor-advisory"
                    ],
                    "url": "https://www.ibm.com/support/pages/node/6841801"
                },
                {
                    "tags": [
                        "vdb-entry"
                    ],
                    "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/240266"
                }
            ],
            "source": {
                "discovery": "UNKNOWN"
            },
            "title": "IBM Cognos Analytics data manipulation",
            "x_generator": {
                "engine": "Vulnogram 0.1.0-dev"
            }
        },
        "adp": [
            {
                "providerMetadata": {
                    "orgId": "96f1cf3c-f80a-4e0f-a063-99a91e3f1325",
                    "shortName": "secretariat-reference",
                    "dateUpdated": "2024-07-04T11:59:33.794Z"
                },
                "references": [
                    {
                        "tags": [
                            "vendor-advisory",
                            "x_cve_prog_secretariat_snapshot"
                        ],
                        "url": "https://www.ibm.com/support/pages/node/6841801/DEMO_2024_06_19"
                    },
                    {
                        "tags": [
                            "vdb-entry",
                            "x_cve_prog_secretariat_snapshot"
                        ],
                        "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/240266/DEMO_2024_06_19"
                    },
                    {
                        "tags": [
                            "vendor-advisory",
                            "x_snapshot"
                        ],
                        "url": "https://www.ibm.com/support/pages/node/6841801/adp-test-6-28"
                    },
                    {
                        "tags": [
                            "vdb-entry",
                            "x_snapshot"
                        ],
                        "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/240266/adp-test-6-28"
                    }
                ]
            }
        ]
    }
}